app/script/conversation/ConversationRepository.js

```diff
@@ -618,8 +618,9 @@ z.conversation.ConversationRepository = class ConversationRepository {
 
   updateConversationFromBackend(conversationEntity) {
     this.conversation_service.get_conversation_by_id(conversationEntity.id).then(conversationData => {
-      this.conversation_mapper.update_properties(conversationEntity, conversationData);
-      this.conversation_mapper.update_self_status(conversationEntity, conversationData);
+      const {name, message_timer} = conversationData;
+      this.conversation_mapper.update_properties(conversationEntity, {name});
+      this.conversation_mapper.update_self_status(conversationEntity, {message_timer});
```

Comment:
@bennyn What we are missing now is a participants list update. People that got removed or added while we were not part of the conversation are not handled.